esque/__init__.py

```diff
@@ -0,0 +1,5 @@
+import configparser
+
+parser = configparser.ConfigParser()
+parser.read("pyproject.toml")
```

Comment:
Is this actually independent from the working directory? Could be that we have to use path relative to `__file__` here.

Reply:
Hm. Yeah it's probably safer to get `__init__` file's path and start from there.